everviz/plugins/_crossplot/_callback/_crossplot_callback.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import plotly.graph_objs as go
```

Comment:
Should we add an entry for plotly in the list of install requirements for everviz? I am sure it is already part of the list of dependencies of already mentioned dependencies but here we are directly using it.